models/engine/file_storage.py

```diff
@@ -58,6 +58,24 @@ def reload(self):
         except:
             pass
 
+    def get(self, cls, id):
+        """Retrieving object by class and/or id
+        """
+        key = cls.__name__ + '.' + id
+
+        if key in self.__objects:
+            return self.__objects[key]
+        else:
+            return None
+
+    def count(self, cls=None):
+        """Return count of objects in storage
+        """
+        if cls:
+            return len(self.all(cls))
```

Comment:
This is technically the only line you need, if you wanted to make your method shorter you could call this because all() takes care if cls was equal to None or not.

Reply:
Thanks, Michelle. Making the change now.